content/it/agile_software_development.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Agile Software Development
```

Comment:
Please consider translate "Agile Software Development" into Italian, if needed!

Ref: "API Gateway" page in pt-br:
- https://github.com/cncf/glossary/pull/479/files
- https://deploy-preview-479--cncfglossary.netlify.app/pt-br/api_gateway/

![image](https://user-images.githubusercontent.com/46767780/155661288-5cd13792-46b0-4887-b492-9c4cc6336ca2.png)